Specs/Number/Japanese/OrdinalModel.json

```diff
@@ -0,0 +1,69 @@
+[
+{
+  "Input": "だい12",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
```

Comment:
Remove the NotSupportedByDesign for python. Python is now supported in the repository. For these new cases, mark both javascript and python as NotSupported only.